voting/tests/common/scenarios.rs

```diff
@@ -127,6 +127,25 @@ fn tx_vote_paper(
     Transaction::new("Create_Vote_Paper".to_owned(), serde_cbor::to_vec(&tx).unwrap())
 }
 
+fn tx_vote(
+    public: &Public,
+    private: &Private,
+    vote_paper_id: H256,
+    choice: VoteChoice,
+    voter_signature: Signature,
```

Comment:
`public` and `private` is not used.,
```suggestion
    vote_paper_id: H256,
    choice: VoteChoice,
    voter_signature: Signature,
```